src/components/forms/DateInput/DateInput.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { DateInput } from './DateInput'
+import { DateInputGroup } from '../DateInputGroup/DateInputGroup'
+import { Fieldset } from '../Fieldset/Fieldset'
+
+export default {
+  title: 'Form/DateInput',
+  parameters: {
+    info: `
+USWDS 2.0 DateInput component
+
+Source: https://designsystem.digital.gov/components/form-controls/#date-input
+`,
+  },
+}
+
+export const defaultDateInput = (): React.ReactElement => (
```

Comment:
nitpick: technically I'd call this a `monthDateInput` instead of "default", and we could in fact have a story for each of `month, day, year`, but this is an enhancement not a blocker.

Reply:
I added additional stories and renamed them :)